openmetadata-service/src/main/java/org/openmetadata/service/resources/services/connections/TestConnectionDefinitionResource.java

```diff
@@ -70,21 +69,19 @@ public void initialize(OpenMetadataApplicationConfig config) throws IOException
     List<TestConnectionDefinition> testConnectionDefinitions =
         repository.getEntitiesFromSeedData(".*json/data/testConnections/.*\\.json$");
 
-    for (TestConnectionDefinition testConnectionDefinition :
-        repository.listAll(EntityUtil.Fields.EMPTY_FIELDS, new ListFilter(Include.ALL))) {
-      repository.delete(ADMIN_USER_NAME, testConnectionDefinition.getId(), true, true);
-    }
-
     for (TestConnectionDefinition testConnectionDefinition : testConnectionDefinitions) {
-      repository.initializeEntity(testConnectionDefinition);
+      repository.prepareInternal(testConnectionDefinition);
+      testConnectionDefinition.setId(UUID.randomUUID());
+      testConnectionDefinition.setUpdatedBy(ADMIN_USER_NAME);
```

Comment:
@mohityadav766 not sure if there's a better way to handle this. Picked this logic up from the `EntityRepository.initializeEntity` implementation